src/common/Utilities.py

```diff
@@ -594,7 +595,7 @@ def is_operator_adjoint(operator, num_tests = 5, max_err = 10e-5, verbose = True
                 print("Pass, with a with normalized error of " + str(norm_err) + " (max: " + str(max_err) + ")")
     return True
 
-
+@pytest.mark.skip(reason="This is a base-class for the actual tests")
```

Comment:
@paskino this line is problematic as it removes all tests of derived classes as well apparently.

Reply:
I believe the easiest is to change the below class name to `DataContainerAlgebraTests`

Reply:
https://github.com/KrisThielemans/SIRF/pull/1

Reply:
I've cherry-picked your commit. it works!